desktop/desktop/lib/cookie-auth/index.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+
+/* eslint no-multi-spaces: [2, { exceptions: { "VariableDeclarator": true } }] */
+
+/**
+ * External Dependencies
+ */
+const ipc = require( 'electron' ).ipcMain;
+const https = require( 'https' );
+const url = require( 'url' );
+const events = require( 'events' );
+
+/**
+ * Module variables
+ */
+const noop = function() {};
+
+function authorize( username, token ) {
+	var responder = new events.EventEmitter();
+	var body = 'log=' + username;
+	var options = url.parse( 'https://wordpress.com/wp-login.php' );
```

Comment:
There's a `login_url` config setting in the main config file that should be used here.